src/adminjs-options.interface.ts

```diff
@@ -349,7 +349,10 @@ export type BrandingOptions = {
    * @new since 6.0.0
    */
   withMadeWithLove?: boolean;
-
+  /**
+   * Optional additional text below the logo/companyName.
+   */
+  additionalText?: string;
```

Comment:
How about just adding `NODE_ENV` here:
https://github.com/SoftwareBrothers/adminjs/blob/master/src/frontend/components/app/version.tsx
if it's just to display the environment information?

We can also add additional option to `branding` but I'd use a different name like `logoTitle`, `logoSubtitle`, `headerText`, `headerMessage`, `menuHeaderMessage`, `menuHeaderText` which would indicate where it'll appear just by looking at it's name.